apps/backend/api/models/Post.js

```diff
@@ -348,20 +348,30 @@ module.exports = bookshelf.Model.extend(Object.assign({
 
   // for event objects, for use in icalendar
   // must eager load the user relation
-  getCalEventData: function (forUserId) {
+  getCalEventData: function (forUserId, eventInvitation) {
     const user = this.relations.user
+
+    const notGoing = eventInvitation.get('response') === EventInvitation.RESPONSE.NO
+    const status = notGoing ? 'CANCELLED' : 'CONFIRMED'
+    const method = notGoing ? 'CANCEL' : 'REQUEST'
+    const ical_sequence = (eventInvitation.get('ical_sequence') || 0) + 1
```

Comment:
seems a little odd to add 1 here as well, why not just use the sequence that's in the database? that makes more sense to me. adding +1 in two places feels brittle to me